bukkit/src/main/java/com/viaversion/viaversion/bukkit/providers/BukkitPickItemProvider.java

```diff
@@ -181,4 +172,54 @@ private int findEmptyHotbarSlot(final PlayerInventory inventory, final int heldS
         }
         return heldSlot;
     }
+
+    private interface BlockToItem {
+        ItemStack apply(final Block block, final boolean includeData);
+
+        static BlockToItem build() {
+            if (PaperViaInjector.hasMethod("org.bukkit.block.data.BlockData", "getPlacementMaterial")) {
+                return (block, includeData) -> {
+                    final ItemStack item = new ItemStack(block.getBlockData().getPlacementMaterial(), 1);
+                    if (includeData && item.getItemMeta() instanceof final BlockStateMeta blockStateMeta) {
+                        blockStateMeta.setBlockState(block.getState());
+                        item.setItemMeta(blockStateMeta);
+                    }
+                    return item;
+                };
+            } else if (PaperViaInjector.hasMethod(Material.class, "isItem")) {
```

Comment:
Okay i found the issue. Some forks have this method backported from modern versions, leading to the Legacy BlockToItem provider not being used even though it should.

Can we use a different detection here? Maybe check if the server version isLess than 1.9?